lib/tasks/ahoy_visit.rake

```diff
@@ -0,0 +1,13 @@
+namespace :ahoy_visit do
+  desc "Clear ahoy visitor count"
+  task clear: :environment do
+    conn = ActiveRecord::Base.connection
+    ActiveRecord::Base.transaction do
+      raise "Must specify PUBLIC_START_DATE" unless ENV["PUBLIC_START_DATE"].present?
+      raise "Prevent from accidentally run in the future" if Date.current > "2021-09-10".to_date
```

Comment:
@radinreth changing to if Rails.production instead of if Date.current > "yyyy-mm-dd"